buku

```diff
@@ -624,7 +624,7 @@ class BukuDb:
 
     @property
     def dbname(self) -> str:
-        return os.path.splitext(os.path.basename(self.dbfile))[0]
+        return os.path.basename(self.dbfile).removesuffix('.db')
```

Comment:
On second thought, we _don't_ want to hide extension unless it's the default one